EIPS/eip-1.md

```diff
@@ -19,29 +20,59 @@ For Ethereum implementers, EIPs are a convenient way to track the progress of th
+
+#### ERC
+
+Application-level standards and conventions, including contract standards such as token standards ([ERC-20](./eip-20.md)), name registries ([ERC-137](./eip-137.md)), URI schemes, library/package formats, and wallet formats.
+
+### Meta EIP
+
+Describes a process surrounding Ethereum or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the Ethereum protocol itself. They may propose an implementation, but not to Ethereum's codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Ethereum development. Any meta-EIP is also considered a Process EIP.
+
+### Informational EIP
+
+Describes an Ethereum design issue, or provides general guidelines or information to the Ethereum community, but does not propose a new feature. Informational EIPs do not necessarily represent Ethereum community consensus or a recommendation, so users and implementers are free to ignore Informational EIPs or follow their advice.
+
+### General Requirements
 
 It is highly recommended that a single EIP contain a single key proposal or new idea. The more focused the EIP, the more successful it tends to be. A change to one client doesn't require an EIP; a change that affects multiple clients, or defines a standard for multiple apps to use, does.
 
 An EIP must meet certain minimum criteria. It must be a clear and complete description of the proposed enhancement. The enhancement must represent a net improvement. The proposed implementation, if applicable, must be solid and must not complicate the protocol unduly.
 
-### Special requirements for Core EIPs
+## Working Groups
 
-If a **Core** EIP mentions or proposes changes to the EVM (Ethereum Virtual Machine), it should refer to the instructions by their mnemonics and define the opcodes of those mnemonics at least once. A preferred way is the following:
+Many proposals will come into EIP process via working groups who take responsibility for their specifications.  Working Group concerns include the technical quality, formatting and notations for their specifications, coordination with other specifications, the workflow for their tasks and documents, and addditional repos for tracking their work.
 
-```
-REVERT (0xfe)
-```
+* Working group proposals first enter the EIP process (see below) as **Drafts**.
+* While WG proposals are **Drafts** their Specification and workflow belongs to the WG.
+* WG EIPs reenter the EIP process to become **Final**.
```

Comment:
```suggestion
* Working group proposals are merged into the EIP repository with a status of `Draft`.
* As long as a working group EIP has a status of `Draft`, every member of the working group should have the same permissions as a co-author.
* With approval from an EIP editor, an EIP's working group can move that EIP's status from `Draft` to `Final`.
```
Is this wording better?

Reply:
A little better, but I really meant that "While WG proposals are **Drafts** their Specification and workflow belongs to the WG."  So it's up to the WG as to whether "every member of the working group should have the same permissions as a co-author."  Also up to the WG what it takes for them approve a PR from Draft to Final.  From there it's up to the Editors what our process is for accepting the PR, and I haven't changed that here.  I'd expect only minor editorial stuff would be appropriate at that point.

Reply:
How about this then:

```
As long as a working group EIP has a status of `Draft`, the working group itself should be treated as if it were the sole author of the EIP for any process described in EIP-1 that requires author approval.
```

Reply:
Hmm.  I think if a WG wants to be a group author then they can without our help.  I can imagine scenarios where an author of an EIP makes a PR of their proposal to Final over the objections of the WG, and we merge it before becoming aware of the controversy, but don't see a need to spell that out here.  If we were to get into the nitty-gritty I'd suggest that at least one WG member should volunteer as an Editor who needs to approve PRs on that WG's EIPs.